components/sendbird/sendbird.app.mjs

```diff
@@ -1,11 +1,233 @@
+    },
+    nextLimit: {
+      type: "integer",
+      label: "Next Limit",
+      max: 200,
+      min: 0,
+      description: "Specifies the number of sent messages to retrieve after `message_ts`. For example, if `message_ts=1484202848298`, then `next_limit=50` returns 50 messages sent from 1484202848299 (message_ts + 1).",
+      optional: true,
+    },
+    include: {
+      type: "boolean",
+      label: "Include",
+      description: "Determines whether to include messages sent exactly on the specified `message_ts` in the results. (Default: **true**)",
+      optional: true,
+    },
+    reverse: {
+      type: "boolean",
+      label: "Reverse",
+      description: "Determines whether to sort the results in reverse chronological order. If set to **true**, messages appear in reverse chronological order where the newest comes first and the oldest last. (Default: **false**)",
+      optional: true,
+    },
+    senderIds: {
+      type: "string[]",
+      label: "Sender Ids",
+      description: "Restricts the search scope to only retrieve messages sent by one or more users with the specified IDs listed in a comma-separated string.",
+      async options({ applicationId }) {
+        return this.listUserOpts(applicationId);
+      },
+      optional: true,
+    },
+    userId: {
+      type: "string",
+      label: "User Id",
+      description: "Specifies the user ID of the sender.",
+      async options({ applicationId }) {
+        return this.listUserOpts(applicationId);
+      },
+    },
+    operatorFilter: {
+      type: "string",
+      label: "Operator Filter",
+      description: "Restricts the search scope to only retrieve messages sent by operators or non-operator users of the channel.",
+      options: constants.OPERATOR_FILTER_OPTS,
+      optional: true,
+    },
+    messageType: {
+      type: "string",
+      label: "Message Type",
+      description: "Specifies a message type to retrieve. If not specified, all messages are retrieved.",
+      options: constants.MESSAGE_TYPE_OPTS,
+      optional: true,
+    },
+    includingRemoved: {
+      type: "boolean",
+      label: "Including Removed",
+      description: "Determines whether to include messages removed from the channel in the results.",
+      optional: true,
+    },
+    withSortedMetaArray: {
+      type: "boolean",
+      label: "With Sorted Meta Array",
+      description: "Determines whether to include the `sorted_metaarray` property in the response.",
+      optional: true,
+    },
+    message: {
+      type: "string",
+      label: "Message",
+      description: "Specifies the content of the message.",
+    },
+    sendPush: {
+      type: "boolean",
+      label: "Send Push",
+      description: "Determines whether to send a push notification for the message to the members of the channel (applicable to group channels only). Unlike text and file messages, a push notification for an admin message is not sent by default. (Default: true)",
```

Comment:
```suggestion
      description: "Determines whether to send a push notification for the message to the members of the channel (applicable to group channels only). Unlike text and file messages, a push notification for an admin message is not sent by default. (Default: true)",
      default: true,
```

Reply:
we cannot do that, because it's going to send the value by default, even tho the property is optional, the API of sendbird can set it as default if it needs, for example, there are properties that cannot be set along another properties, in this case, not set the property is not the same as set an default value even tho it's the same value that they say as its default.